eos/form-factors/Makefile.am

```diff
@@ -146,6 +148,8 @@ parametric_bsz2015_TEST_SOURCES = parametric-bsz2015_TEST.cc
 
 parametric_bmrvd2022_TEST_SOURCES = parametric-bmrvd2022_TEST.cc
 
+parametric_fvdv2018_TEST_SOURCES = parametric-bmrvd2022_TEST.cc
```

Comment:
Copy & paste error?

Reply:
To be clear, this comment applies to ``parametric_fvdv2018_TEST_SOURCE``, which uses the ``-bmrvd2022_TEST.cc``.

Reply:
Oh! Thanks!